CMakeLists.txt

```diff
@@ -716,8 +733,8 @@ if(AVIF_BUILD_APPS)
         )
     endif()
     if(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
-        include(LocalLibargparse)
-        if(TARGET libargparse)
+        if(AVIF_LOCAL_LIBARGPARSE)
```

Comment:
Why not do `include(LocalLibargparse)` and assume `AVIF_LOCAL_LIBARGPARSE` is `ON` by default?

Reply:
That makes sense. I suppose it's implied by `AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP`.